ansible/roles/stack-sunbird/templates/inbound.env

```diff
@@ -1,25 +1,38 @@
-campaign.url=http://uci-service:9999
+#External Services
+CAMPAIGN_URL=http://uci-service.{{namespace}}.svc.cluster.local:9999
 
-#Kafka Configuration
-spring.kafka.bootstrap-servers={{sunbird_processing_kafka_host}}
-spring.kafka.consumer.auto-offset-reset=earliest
-spring.kafka.consumer.value-deserializer=org.springframework.kafka.support.serializer.JsonDeserializer
-spring.kafka.producer.value-serializer=org.springframework.kafka.support.serializer.JsonSerializer
-schema.registry.url={{zookeepers}}
-inbound-error={{env_name}}.inbound-error
-inboundProcessed={{env_name}}.inbound-processed
-inboundUnprocessed={{env_name}}.inbound-unprocessed
-campaign={{env_name}}.campaign
+#Kafka
+BOOTSTRAP_SERVERS={{sunbird_processing_kafka_host}}
+KAFKA_INBOUND_PROCESSED_TOPIC={{env_name}}.inbound-processed
+KAFKA_CAMPAIGN_TOPIC={{env_name}}.campaign
+KAFKA_INBOUND_UNPROCESSED_TOPIC={{env_name}}.inbound-unprocessed
+KAFKA_INBOUND_GS_OPTED_OUT_TOPIC={{env_name}}.gs-opted-out
+KAFKA_INBOUND_ERROR_TOPIC={{env_name}}.inbound-error
+KAFKA_OUTBOUND_TOPIC={{env_name}}.outbound
+KAFKA_TELEMETRY_TOPIC={{env_name}}.uci.telemetry
+KAFKA_ODK_TRANSFORMER_TOPIC={{env_name}}.com.odk.transformer
+KAFKA_ODK_TRANSFORMER_TOPIC_PATTERN={{env_name}}.com.odk.*
 
-# Cassandra Configuration
-spring.data.cassandra.contactpoints={{sunbird_cassandra_host}}
-spring.data.cassandra.port=9042
-spring.data.cassandra.keyspace-name={{env}}_uci_store
-spring.data.cassandra.pool.heartbeat-interval=30s
-spring.data.cassandra.pool.max-queue-size=2000
-spring.data.cassandra.pool.idle-timeout=10s
-spring.data.cassandra.pool.pool-timeout=1s
+#FormsDB
+FORMS_DB_URL=postgresql://{{uci_postgres_host}}:5432/{{uci_forms_postgres_database}}
+FORMS_DB_USERNAME={{uci_postgres_user}}
+FORMS_DB_PASSWORD={{uci_postgres_password}}
+FORMS_DB_NAME={{uci_forms_postgres_database}}
+FORMS_DB_HOST={{uci_postgres_host}}
+FORMS_DB_PORT=5432
 
-# Service Configuration
-server.port=8085
-logging.level.root=INFO
\ No newline at end of file
+#Cassandra
+CASSANDRA_URL={{sunbird_cassandra_host}}
+CASSANDRA_PORT=9042
+CASSANDRA_KEYSPACE={{env}}_uci_store
+
+#Ports
+INBOUND_INTERNAL_PORT=8085
+
+#ODK
+ODK_URL=https://agg.staging.saksham.samagra.io/
+ODK_USER=test
+ODK_PASS=abcd1234
```

Comment:
No user and pass in public repos

Reply:
Using the URL and the creds I am able to login to the system. If I can do things on the system, nobody will even know who did it

Reply:
Can you please the following variables to private repo, I have made changes to use those variables here.
ODK_USER={{uci_odk_user}}
ODK_PASS={{uci_odk_password}}
uci_odk_user=administrator
uci_odk_password=administrator

Reply:
I have added these vars in private. Going forward you need to add the vars to dev env and test it out. Please get access for the same

Reply:
Sure, Thank you!